instagram/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:17-alpine
+
+WORKDIR /usr/app
+
+RUN mkdir -p /usr/app/instagram
```

Comment:
I find this a bit confusing. Just use `/usr/app` as the root folder. No need for copying things in batches. Just copy everything into this folder and run `npm install`. This also avoids the need for doing a separate `WORKDIR`.